roles/gateway/defaults/main.yml

```diff
@@ -5,3 +5,7 @@
 secrets_repo:
   name: UNDEFINED
   url: null
+
+openvpn_server_name: "server"
```

Comment:
I know you don't need quotes here; not sure if they do any damage, but it's not the prevailing style, so, should remove them